arches_lingo/src/arches_lingo/pages/SchemeList.vue

```diff
@@ -1 +1,44 @@
-<template>Schemes list placeholder</template>
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+import SchemeCard from "@/arches_lingo/components/scheme/SchemeCard.vue";
+import { fetchSchemes } from "@/arches_lingo/api.ts";
+
+import type { SchemeResource } from "@/arches_lingo/types";
+
+const schemes = ref<SchemeResource[]>([]);
+
+onMounted(async () => {
+    schemes.value = await fetchSchemes();
+    schemes.value.unshift({
+        resourceinstanceid: "placeholder-id",
```

Comment:
I wasn't exactly sure what to do, which is why I ended up punting with `placeholder-id`... My thought was that a PR for adding the scheme-editor and it's corresponding route would be coming hot on the heels of this, and we could just swap `placeholder-id` for the new scheme route. 

As you point out, it's probably not desirable to have a dummy uuid in lieu of string `placeholder-id` because we'd need to handle that on the back-end. 

Alternatively, I could mint the new scheme resourceid here - but I see that as less desirable since it would create a new uuid each time `/schemes` is visited.

All that to say, I'm not exactly sure what to do about this

Reply:
Ah, got it. If we're about to swap in a new route for the scheme editor then all is good 👍